src/main/java/org/kiwiproject/dropwizard/error/config/CleanupConfig.java

```diff
@@ -24,12 +24,32 @@
     }
 
     /**
-     * The strategy to use for what to cleanup. Defaults to {@link CleanupStrategy#RESOLVED_ONLY}
+     * The strategy to use for what to cleanup. Defaults to {@link CleanupStrategy#ALL_ERRORS}
      */
-    private CleanupStrategy cleanupStrategy = CleanupStrategy.RESOLVED_ONLY;
+    private CleanupStrategy cleanupStrategy = CleanupStrategy.ALL_ERRORS;
 
     /**
-     * The duration of time that an error will live before being deleted.
+     * The duration of time that a resolved error will live before being deleted.
```

Comment:
```suggestion
     * The duration that a resolved error will live before being deleted.
```

Since duration refers to time by definition, the phrase "of time" seems redundant. I think me taking Spanish lessons on Duolingo is making me pay (too much) attention to this stuff...  :thinking: